connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -106,7 +106,7 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
 
   // Mapping from relation ID (passed to client) to runtime dataframe. Used for callbacks like
   // foreachBatch() in Streaming. Lazy since most sessions don't need it.
-  private lazy val dataFrameCache: ConcurrentMap[String, DataFrame] = new ConcurrentHashMap()
+  private[spark] lazy val dataFrameCache: ConcurrentMap[String, DataFrame] = new ConcurrentHashMap()
```

Comment:
Can you update the doc?

Reply:
Sure, this is same as https://github.com/apache/spark/pull/46570#discussion_r1600950566 FWIW.